docs/source/services/timelock_service/migration.rst

```diff
@@ -0,0 +1,145 @@
+.. _timelock-migration:
+
+Migration to External Timelock Services
+=======================================
+
+.. danger::
+
+   Improperly configuring one's cluster to use external timestamp and lock services can result in **SEVERE DATA
+   CORRUPTION**! Please contact the AtlasDB team before attempting a migration.
+
+Why Migration?
+--------------
+
+AtlasDB assumes that timestamps returned by the timestamp service are monotonically increasing. In order to preserve
+this guarantee when moving from an embedded timestamp service to an external timestamp service, we need to ensure
+that timestamps issued by the external timestamp service are larger than those issued by the embedded one.
+Otherwise, this can lead to serious data corruption.
+
+The migration process must be run offline (that is, with no AtlasDB clients running during migration) and basically
```

Comment:
We should be explicit in the steps below that "Step 0: Shutdown your AtlasDB clients". We don't want people to forget that, grab a timestamp and then end up breaking the timestamp server guarantees when they restart their client.